### PR TITLE
Update DevGuide and UserGuide (UI Class diagram + Select + Close Command)

### LINKED ISSUE #133: Update UserGuide (Select and Close Command)

### LINKED ISSUE #134: Update DevGuide (Logic - UI relationship)

### PR DESCRIPTION
**Pull request by daongochieu2810**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -232,12 +232,27 @@ Format: `deleteC INDEX`
 Examples:
 * `calories` followed by `deleteC 2` deletes the 2nd recipe in Consumption List.
 
-### Clearing all consumption : `clearC'
+### Clearing all consumption : `clearC`
```
Comment (TianYong-Goh) [inline docs/UserGuide.md:306]: Thank you for helping me to change this.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -232,12 +232,27 @@ Format: `deleteC INDEX`
...
+### Open a single recipe : `selectR`
+
+Display a single recipe in full view.
+
+Format: `selectR INDEX`
+
+Examples:
+* `selectR 1` shows the 1st recipe in full view (left drawer).
+
+### Close left drawer : `close`
```
Comment (TianYong-Goh) [inline docs/UserGuide.md:321]: Can try to rephase it? cause I don't think the user know what is a drawer. Maybe "Close the recipe full view" ?

Comment (oliviajohansen) [reply docs/UserGuide.md:321]: another option is adding drawer to glossary

**Review by oliviajohansen — APPROVED** [review]
lgtm

**Review by caitlinjee — APPROVED** [review]
lgtm, agree with the suggestion to add drawer to glossary

**Review by jiaqi20 — APPROVED** [review]
lgtm

**Review by TianYong-Goh — APPROVED** [review]

**Pull request merged**